.github/workflows/release.yml

```
@@ -21,6 +21,8 @@ jobs:
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN_AENEASR }}" > ~/.npmrc
       - run: |
           npx lerna version --no-private --no-git-tag-version --no-push -y ${{ github.event.release.tag_name }}
+      - run: |
+          make format
```

You should be able to put this one one line.

```suggestion
      - run: make format
```

---

Yup, copied from above :)